hoi/metrics/gradient_oinfo.py

```diff
@@ -48,9 +47,7 @@ class GradientOinfo(HOIEstimator):
     __name__ = "Gradient O-Information"
 
     def __init__(self, x, y, multiplets=None, verbose=None):
-        HOIEstimator.__init__(
-            self, x=x, y=None, multiplets=multiplets, verbose=verbose
-        )
+        HOIEstimator.__init__(self, x, None, multiplets, verbose)
```

Comment:
This is a bit dangerous to write it like that. It would be better to keep the optional arguments. Imagine that we change the order in the `HOIEstimator.__init__`. Can you put back `x=x, y=None, multiplets=multiplets, verbose=verbose`?